src/core/model.js

```diff
@@ -334,13 +335,15 @@ export default class Model extends Object3D {
     return this;
   }
 
-  render(uniforms = {}, attributes = {}, parameters = {}, samplers = {}) {
+  render(uniforms = {}, attributes = {}, samplers = {}) {
```

Comment:
`parameters` was never part of documented API, so removing here is fine. We only documented `uniforms`.

Can you also update the doc, to include `attributes` and `samplers`. Also in `whats-new.md`.